src/telemetry/telemetry.bif

```diff
@@ -548,3 +558,36 @@ function Telemetry::__dbl_histogram_sum%(val: opaque of dbl_histogram_metric%):
 		return zeek::make_intrusive<DoubleVal>(0.0);
 		}
 	%}
+
+
+function Telemetry::__collect_metrics%(prefix: string, name: string%): any_vec
+	%{
+	auto metrics = telemetry_mgr->CollectMetrics(sv(prefix), sv(name));
+
+	static auto metrics_vector_type = zeek::id::find_type<VectorType>("any_vec");
+	auto vec = zeek::make_intrusive<VectorVal>(metrics_vector_type);
+
+	for ( const auto& m : metrics )
+		{
+		auto rec = m.GetMetricRecord();
+		vec->Append(rec);
```

Comment:
Is there a particular reason to stretch this out over multiple lines? Could just be:

```suggestion
		vec->Append(m.GetMetricRecord());
```